.github/workflows/update-flake.yml

```
@@ -72,19 +72,35 @@ jobs:
 
       - name: create pull request
         env:
+          GH_REPOSITORY: ${{ github.repository }}
+          GH_REPO_OWNER: ${{ github.repository_owner }}
           GH_TOKEN: ${{ steps.generate-token.outputs.token }}
           base_branch: ${{ matrix.branch }}
+          body: "This is an automated update triggered by the [workflow run #${{ github.run_id }}](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})."  # yamllint disable-line rule:line-length
+          label: "topic: dependencies"
           pr_branch: update_flake_lock_action_${{ matrix.branch }}
           title: "${{ startsWith(matrix.branch, 'release') && format('[{0}] ', matrix.branch) || '' }}flake: update public and dev inputs"  # yamllint disable-line rule:line-length
         run: |
           git switch --create "$pr_branch"
+          git push origin "$pr_branch" --force --set-upstream
 
-          git push origin "$pr_branch" \
-            --force \
-            --set-upstream
+          if (($(
+            gh api \
+              --field per_page=1 \
+              --jq length \
+              --method GET \
+              --raw-field head="$GH_REPO_OWNER:$pr_branch" \
+              "/repos/$GH_REPOSITORY/pulls"
```

Looks like you've sorted lexicographically, but for me I'd prefer to group fields together logically and sort them more imperatively...

E.g. "GET URL FLAGS, then post-process using jq"


```suggestion
            gh api \
              --method GET \
              "/repos/$GH_REPOSITORY/pulls" \
              --field per_page=1 \
              --raw-field head="$GH_REPO_OWNER:$pr_branch" \
              --jq length
```


```suggestion
            gh api "/repos/$GH_REPOSITORY/pulls" \
              --method GET \
              --field per_page=1 \
              --raw-field head="$GH_REPO_OWNER:$pr_branch" \
              --jq length
```


```suggestion
            gh api -X GET "/repos/$GH_REPOSITORY/pulls" \
              --field per_page=1 \
              --raw-field head="$GH_REPO_OWNER:$pr_branch" \
              --jq length
```

---

Yes, this makes it much more readable. Sorry for the oversight.